### DISCUSSION TITLE
Why can't we you use the NextBSD or Darwin OS.

### DISCUSSION THREAD

**Discussion by SunilSherkhane** [ravynsoft/ravynos#368] [General]
Why can't you use the NextBSD or Darwin OS as Darwin was open-sourced by Apple and NextBSD was half Baked OS?

**Comment by theotheroracle** [comment]
that's in the faq, it's hardware support

**Reply by mszoek** [reply]
> Why can't you use the NextBSD or Darwin OS as Darwin was open-sourced by Apple and NextBSD was half Baked OS?

What makes you think I'm not using those sources? NextBSD did great work to bring in Mach and parts of Darwin (launchd, launchctl, libxpc, etc), but it was stale and against a fairly old FreeBSD base (10.x). I took that work and brought it up to date for FreeBSD-CURRENT, and have picked up where they left off. We also include a growing list of Darwin and Apple open-source projects.

**Reply by SunilSherkhane** [reply]
Thanks... I am waiting for the ISO which will work on my aged system.
